Cubical/Algebra/Ring/Properties.agda

```diff
@@ -29,11 +29,11 @@ private
   that should become obsolete or subject to change once we have a
   ring solver (see https://github.com/agda/cubical/issues/297)
 -}
-module Theory (R' : Ring {ℓ}) where
+module Theory (R : Ring {ℓ}) where
 
-  open Ring R' renaming ( Carrier to R )
```

Comment:
Why do this change? I kind of prefer to omit the angle brackets around R...

Reply:
I'll revert that - I don't have a strong preference and I don't remember why I changed that.

Reply:
This should get better once we go for partially bundled structures as suggested by: https://github.com/agda/cubical/issues/405

Reply:
Good!